src/components/utils/Input/InputElement.tsx

```diff
@@ -30,16 +39,20 @@ const InputElement = React.forwardRef<HTMLInputElement | HTMLTextAreaElement, In
         _disabled={{
           opacity: 1, // we have nested disabled elements, so we don't want lower opacities to multiply
         }}
-        _placeholder={{
-          opacity: 0,
-          color: 'gray-50',
-          transition: 'opacity 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms',
-        }}
-        _focus={{
-          '::placeholder': {
-            opacity: 0.4,
-          },
-        }}
+        _placeholder={
+          !standalone && {
```

Comment:
TS might warn you here (after my PR gets merged). 

Ideally you'd  want `!standalone ? styles : {}` instead of `!standalone && styles`

Reply:
Done!